api/pom.xml

```diff
@@ -29,5 +29,11 @@
 			<scope>provided</scope>
 			<version>${fhirVersion}</version>
 		</dependency>
+		<dependency>
```

Comment:
I think this is unnecessary (See one dependency above).

Reply:
Yes, you have right. I corrected it.